main.py

```diff
@@ -1,3 +1,4 @@
 if __name__ == "__main__":
     print("Hello prowl!")
+    print("Requires review")
```

Comment:
This needs changing here

Reply:
This comment should not overwrite my approval below